client/src/pages/LogIn/LogIn.js

```diff
@@ -10,15 +11,17 @@ class LogIn extends Component {
 
     handleInputChange = event => {
         const { name, value } = event.target;
-        this.setState({[name]: value}, this.callback);
+        this.setState({[name]: value});
       };
 
     handleFormSubmit = event => {
         event.preventDefault();
         if (this.state.password !== this.state.confirmPassword){
```

Comment:
Should be using API to call the login function, password should be returned in the response